packages/aws-otel-collector/aws-otel-collector.spec

```
@@ -0,0 +1,78 @@
+%global goproject github.com/aws-observability
+%global gorepo aws-otel-collector
+%global goimport %{goproject}/%{gorepo}
+
+Name: %{_cross_os}aws-otel-collector
+Version: 0.40.0
+Release: 1%{?dist}
+Summary: An AWS supported service to send telemetry data
```

This is not a service and the "AWS supported" statement is confusing in this context. I would use the summary from the GitHub project page:
```suggestion
Summary: AWS Distro for OpenTelemetry Collector
```